src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -241,6 +248,14 @@ private void updateOutOfSyncInvoice(Invoice invoice) {
     });
   }
 
+  private CompletableFuture<Boolean> recalculateTotals(Invoice updatedInvoice, Invoice invoiceFromStorage) {
```

Comment:
As for method returns boolean value, probably it is better to name it something like `isTotalsRecalculationRequired(..)` for better code readability.